apps/heft/src/plugins/TypeScriptPlugin/TypeScriptPlugin.ts

```diff
@@ -237,6 +237,8 @@ export class TypeScriptPlugin implements IHeftPlugin {
     if (tsconfigPaths.length === 0) {
       // If there are no TSConfigs, we have nothing to do
       return;
+    } else {
+      buildProperties.isTypeScriptProject = true;
```

Comment:
```suggestion
      buildProperties.isTypeScriptProject = false;
      return;
    } else {
      buildProperties.isTypeScriptProject = true;
```

I see this property as having three values: `undefined` means "too early to tell", and then `true` and `false` mean "is/is not TypeScript" respectively.